test/libsolidity/syntaxTests/types/address/type_type_to_address.sol

```diff
@@ -28,19 +28,8 @@ contract C {
     }
 }
 // ----
+// UnimplementedFeatureError: Not yet Implemented - FixedPointType.
```

Comment:
I'll just comment out `fixed` in this test.

Reply:
Why would you like to comment out `fixed` in this test?

Reply:
Because I think the rest of the test makes sense, and this throw is making some other errors.